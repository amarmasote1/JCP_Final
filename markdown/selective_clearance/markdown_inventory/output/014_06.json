{
    "message": "Success",
    "body": [
        {
            "partiallyReservedQty": 0,
            "shortageQty": 0,
            "orgCode": "JCP",
            "reservedQty": 3,
            "requestLine": {
                "deliveryMethod": null,
                "channel": null,
                "lineID": "1",
                "storeID": "2920",
                "itemID": "7063000-3024",
                "schedule": false,
                "uom": "EACH",
                "shippable": true,
                "reservationID": "30564ef1-6b42-46a2-a45b-18f8a516c726",
                "procurementAllowed": false,
                "qty": 3,
                "bopus": false,
                "increaseCapacity": null,
                "shutdown": false
            },
            "problemType": "SUCCESS"
        }
    ],
    "status": 200
}