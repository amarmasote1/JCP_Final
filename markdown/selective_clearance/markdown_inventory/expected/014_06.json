{
	"status": 200,
	"message": "Success",
	"body": [{
		"shortageQty": 0,
		"reservedQty": 3,
		"partiallyReservedQty": 0,
		"problemType": "SUCCESS",
		"orgCode": "JCP",
		"requestLine": {
			"lineID": "1",
			"itemID": "7063000-3024",
			"uom": "EACH",
			"bopus": false,
			"shippable": true,
			"deliveryMethod": null,
			"increaseCapacity": null,
			"shutdown": false,
			"qty": 3,
			"storeID": "2920",
			"channel": null,
			
			"procurementAllowed": false,
			"schedule": false
		}
	}]
}